### PR TITLE
make DM_SLAVE_MMS behaviour similar to MD_DEVICES, always returns an …

### PR DESCRIPTION
**Pull request by tanabarr**
…array

Signed-off-by: Tom Nabarro <tom.nabarro@outlook.com>

### CODE REVIEW

File: IML/DeviceScannerDaemon/EventTypes.fs [hunk 1]
```diff
@@ -73,6 +73,12 @@ let private parseMdDevices x =
     |> Map.filter (fun (k:string) _ -> k.EndsWith("_DEV"))
     |> Map.toArray
     |> Array.map (snd >> unwrapString)
+let private parseDmSlaveMms (x:Map<string,Json>) =
+  x
+    |> Map.filter (fun (k:string) _ -> k.Equals("IML_DM_SLAVE_MMS"))
```
Comment (jgrund) [inline IML/DeviceScannerDaemon/EventTypes.fs]: Should be able to do this without an `Equals` method call.

`Map.filter ((=) "IML_DM_SLAVE_MMS")`

Comment (tanabarr) [reply IML/DeviceScannerDaemon/EventTypes.fs]: nice, thanks

Comment (jgrund) [reply IML/DeviceScannerDaemon/EventTypes.fs]: ah, right. 

`Map.filter (fst >> ((=) "IML_DM_SLAVE_MMS"))`

Comment (tanabarr) [reply IML/DeviceScannerDaemon/EventTypes.fs]: tried that

**Pull request merged**